ldcpy/plot.py

```diff
@@ -139,10 +141,21 @@ def get_metrics(self, da):
         da_data = da
         da_data.attrs = da.attrs
 
+        # lat/lon dim names are different for ocn and atm
+        if da_data.dims[0] == 'time':
+            lat_dim = da_data.dims[1]
+            lon_dim = da_data.dims[2]
+        else:
+            lat_dim = da_data.dims[0]
+            lon_dim = da_data.dims[1]
+
+        # print(lat_dim, lon_dim)
+
```

Comment:
It looks like time is assumed to be the first (or last) dimension specified. Is there any chance that it could be the second dimension (or that another dimension like "level" might be the first dimension?

Reply:
Thanks - I will modify this to be more generic.

Reply:
I am going to go ahead and merge this - I still have other mods to make for the ocean in plot.py, so I will address this then.